pkg/cli/lint.go

```diff
@@ -47,69 +41,32 @@ func Lint() *cobra.Command {
 		Args:    cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
-
-			// Cheap and dirty way to handle duplicates
-			linterSet := map[string]struct{}{}
-
-			// Get all default linters, ignoring disabled ones
-			for _, e := range linter_defaults.GetDefaultLinters(linter_defaults.LinterClassApk) {
-				if !slices.Contains(disabled, e) {
-					linterSet[e] = struct{}{}
-				}
-			}
-
-			// Enable non-default lints
-			for _, e := range enabled {
-				if !slices.Contains(disabled, e) {
-					linterSet[e] = struct{}{}
-				}
+			g, ctx := errgroup.WithContext(ctx)
+			g.SetLimit(runtime.GOMAXPROCS(0))
+
+			errs := []error{}
+			var mu sync.Mutex
```

Comment:
I have a slight preference for avoiding the mutex by doing `make([]error, len(args))` and having each goroutine set `errs[i] = err` then letting `errors.Join` filter out the nils.

Reply:
Ehhh, that feels clever for a very very small performance gain. If you find out it's slowing us down we can do your thing.